CMakeLists.txt

```diff
@@ -34,14 +34,21 @@ else()
     message(FATAL_ERROR "Boost not found!")
 endif()
 
-# Add executable
-add_executable(lmkdb ${SOURCE_FILES})
+add_executable(lmkdb ${SOURCE_FILES}) # Add executable
+target_link_libraries(lmkdb PUBLIC Boost::filesystem) # Link Boost libraries
+target_include_directories(lmkdb PUBLIC ${INCLUDE_PATHS} ${Boost_INCLUDE_DIRS}) # Include directories for the target
 
-# Link Boost libraries
-target_link_libraries(lmkdb PUBLIC Boost::filesystem)
+# readline library
+find_path(READLINE_INCLUDE_DIR readline/readline.h PATHS /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include)
+find_library(READLINE_LIBRARY readline PATHS /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/lib)
```

Comment:
my readline library is stored in `/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/lib`. I assume yours are as well

Reply:
same for us I think